config/values/00-values.yml

```diff
@@ -89,3 +89,10 @@ blobstore:
   resource_directory_key: "cc-resources"
   buildpack_directory_key: "cc-buildpacks"
   aws_signature_version: "2"
+
+#! configuration for private registry
+private_registry:
```

Comment:
Could you please rename this to `system_registry`?

Reply:
done